parser/parse.go

```diff
@@ -260,6 +260,26 @@ func (b *Builder) AddDirTo(dir string, u *types.Universe) error {
 	return b.findTypesIn(canonicalizeImportPath(b.buildPackages[dir].ImportPath), u)
 }
 
+func (b *Builder) CanonicalPath(dir string) (string, error) {
```

Comment:
Returning the `package` is not going to help me. I need the `canonicalizeImportPath(package.ImportPath)`.